lib/main.dart

```diff
@@ -1,19 +1,60 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:kahootify/core/data/shared_preferences_repository.dart';
+import 'package:kahootify/features/splash/views/splash_page.dart';
 import 'package:kahootify/features/welcome/views/welcome_page.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+import 'core/bloc/settings_cubit.dart';
+
+class SimpleBlocObserver extends BlocObserver {
```

Comment:
SimpleBlocObserver looks like a class from w3schools tutorial, the name should determine usage, that's how you write self documenting code